rs/core/corpora_cli/src/context.rs

```diff
@@ -0,0 +1,62 @@
+use std::env;
+
+use corpora_client::apis::configuration::Configuration;
+
+pub struct Context {
+    pub config: Configuration,
+}
+
+impl Context {
+    /// Initialize a new context with OAuth authentication and configuration
+    pub fn new(server_url: &str) -> Self {
```

Comment:
instead of taking the server url as an argument we should get it from .corpora.yaml or default. (app:8877 in dev, and maybe corpora.site for release (?))

`config` should maybe be `api_config` because I think we will also have corpora config. Don't forget about `.corpora/.id` (or better).

Reply:
still need to write the .id for later methinks